base/special/trig.jl

```diff
@@ -1073,6 +1073,26 @@ end
 
 tand(x::Real) = sind(x) / cosd(x)
 
+"""
+    sincosd(x)
+
+Simultaneously compute the sine and cosine of `x`, where `x` is in degrees.
+
+"""
+function sincosd(x::Real)
+    if isinf(x)
```

Comment:
Isn't these conditions checked in `sind` and `cosd`? Why check here?

Reply:
Because without it, then the user will see an error on the function `sind` if `Inf` or `NaN` is passed. I though it could lead to confusion.

Reply:
I don't think we do error checking like that elsewhere though.